mlir/utils/performance/perfRegressionReport.py

```diff
@@ -120,6 +121,8 @@ def getPerfDate(statsPath: PurePath, default="???"):
         oldLabel = "copy"
 
     data, summary = computePerfStats(oldDf, newDf, oldLabel, newLabel)
+    oldNewPaths = {"old" : os.path.abspath(oldDataPath),
```

Comment:
On the one hand, this is why we have `perf-run-date`, so you can mark which file is which (so long as they're in separate directories).
This also seems like it'll add useless noise to the CI-generated regression reports.

I'm not opposed, but I figured I'd point this out.

Reply:
Yeah, I didn't think at the noise in the CI. Let's do this, I will close this for now and see if I can create multiple folders where I store my local data (so that csv, tsv and reports are connected, which is what I really need)